osx/build.sh

```diff
@@ -368,6 +369,17 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+((STEP++))
+echo "Updating KA Lite installer version"
+KA_LITE_VERSION="$(kalite --version)"
+DOCS_PATH="$SCRIPTPATH/KA-Lite-Packages/docs"
+sed -i '' "13s/0.17.0/$KA_LITE_VERSION/g" "$DOCS_PATH/INTRODUCTION.rst.rtfd/TXT.rtf"
+sed -i '' "12s/0.17.0/$KA_LITE_VERSION/g" "$DOCS_PATH/SUMMARY.rtfd/TXT.rtf"
+sed -i '' "12s/0.17.0/$KA_LITE_VERSION/g" "$DOCS_PATH/README.rst.rtf"
+if [ $? -ne 0 ]; then
```

Comment:
This only tests the output of the previous command failed.

You can do the following to test all of it:

```bash
sed ... &&
sed ... &&
sed ... ||
(echo "failed to update installer version" && exit -1)
```